source/reference/limitations.txt

```diff
@@ -25,6 +25,9 @@ General Limitations
 -------------------
 
 - The minimum supported server version is MongoDB 6.0.
+- ``mongosync`` supports major MongoDB releases such as 6.0, 7.0. Rapid
+  releases such as 6.1, 6.2 are not supported. For more information on
+  MongoDB versioning, see :ref:`release-version-numbers`.
```

Comment:
[optional suggestion]

Because this is in a section on Limitations, consider leading with the limitation aspect of this bullet. As in something like:

> ``mongosync`` does not support (synchronizing to or from?) rapid release versions, such as 6.1 and 6.2. ``mongosync`` only supports major MongoDB releases such as 6.0, 7.0. For more information on
  MongoDB versioning, see :ref:`release-version-numbers`.